### DISCUSSION TITLE
Interactive content for Sunbird ED knowledge sharaing

### DISCUSSION THREAD

**Discussion by ashokreddy1208** [Sunbird-Ed/Community#138] [Documentation] [answered]
@luckynvdu @sharathkashyap 

Can you please provide more information on below two tickets which are part of 4.10.3 Sunbird-ED release.

What is the functional impact etc.,

https://project-sunbird.atlassian.net/browse/SB-30675
https://project-sunbird.atlassian.net/browse/SB-30530

cc :  @anandvarada @kameshbhr @mmanimegalai

**Comment by sharathkashyap** [comment, marked as answer]
@ashokreddy1208 The Artefacts are attached as part of ticket.
cc: @luckynvdu
